### ISSUE TITLE
The HDRI background always turn pink

### ISSUE THREAD

**Issue by Permork72** [Poly-Haven/polyhavenassets#16]
I believe I have installed the Poly Haven addon correctly. I have set up the file path for the assets, and it shows up in the assets browser no prob. 

But when I drag an HDRI in to the scene, the background is just pink. All other assets works just fine - textures and objects.

Do you have an insight to why this is happening? I would really love to have easy access to all the HDRIs here.

**Comment by gregzaal** [comment]
I can't reproduce this issue here so I'll need more information to try and help you.

1. First please make sure you're using the latest version of the add-on (there's an update button in the preferences) and the latest version of Blender.
2. Then delete your asset library folder.
3. Run Blender from a command prompt/terminal, and click the fetch button again.
4. Check the command prompt to see if there are any errors, and paste the full output here.
5. If there are no errors, try add an HDRI to your scene, and check the command prompt again.
6. Finally, look at the HDRI image itself in the Image Editor in Blender, and in the right side panel check the image path and copy paste that here.

**Comment by Permork72** [comment]
Hi again. 

I have now done as you said, opened Blender with command window and did a new fetch of all assets.

And this time it all seems to work as intended. And no error showed up on command promt.

I think part of the problem  (and what I probably should have mentioned here)  may be that I moved my asset folder to another location a while ago. I thought it was enough to only re-adjust the path to the new location. All the textures and objects worked fine after that. I only tried out the HDRIs later.

If this was the error, it may be prudent to warn users against moving their folder after installation?

Thank you for helping me with this. I am grateful :)
